examples/Passwordless.Example/Passwordless.Example.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Passwordless" Version="1.0.2" />
+    <ProjectReference Include="..\..\src\Passwordless\Passwordless.csproj" />
```

Comment:
I switched this to a local reference while debugging/testing and was wondering if we should keep it this way? Apparently the Identity example already uses a local reference. It's quite convenient to test things out using a version that hasn't been released yet.

Reply:
Agreed on the testing convenience. Can't think of a reason to not do it.  👍